change/@internal-acs-ui-common-80281123-61cb-4c24-abc9-eb983167b96f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
These changelogs should be type "patch".

Reply:
Auto merge was on :(